regulation-worker/internal/delete/api/api.go

```diff
@@ -49,11 +53,39 @@ func (api *APIManager) Delete(ctx context.Context, job model.Job, destConfig map
 	}
 	req.Header.Set("Content-Type", "application/json")
 
+	var tokenStatusCode int
+	var accountSecretInfo *oauth.AuthResponse
+	// identifier to know if the destination supports OAuth for regulation-api
+	rudderUserDeleteAccountId, delAccountIdExists := destDetail.Config["rudderDeleteAccountId"]
```

Comment:
To increase the readability of the code.
```suggestion
	rudderUserDeleteAccountId, isOAuthRequired := destDetail.Config["rudderDeleteAccountId"]
```

Reply:
Done